packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -48,6 +48,9 @@ import { carbonElement as customElement } from '../../internal/vendor/@carbon/we
 
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
+// This matches the CSS transition timing.
+const animationTiming = 240;
```

Comment:
See https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/src/components/carousel/carousel.ts#L15 and https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/src/components/carousel/carousel.ts#L164 for an example of getting this value from [the relevant Carbon package](https://github.com/carbon-design-system/carbon/blob/main/packages/motion/src/index.js#L11).

Reply:
Thank you!!! I tried finding this and wasn't able to. So helpful.